readme.md

```diff
@@ -725,73 +725,92 @@ vrf keepalive
 
 ```
 
-#### Generate Switch Config With Overrides
+#### Generate Switch Config With Custom Config Injection
 
-This option allows you to pass in a file that contains switch configuration that CANU will ignore on config generation. A use case would be to ignore the site connection on spine01, or an edge device that CANU does not recognize.
+Pass in a switch config file that CANU will inject into the generated config. A use case would be to add custom site connections.
```

Comment:
This option allows extension and maintenance of switch configurations beyond plan-of-record.  A YAML file expresses custom configurations across the network and these configurations are merged with the plan-of-record configurations.

WARNING:  Extreme diligence should be used applying custom configurations which override plan-of-record generated configurations.  Custom configurations will _overwrite_ generated configurations!  Override/overwrite is by design to support and document cases where site-interconnects demand "nonstandard" configurations or a bug must be worked around.